### ISSUE TITLE
[Episode 3] Unable to create/join a game on the proxy server

### ISSUE THREAD

**Issue by Matt-Swift** [fuzziqersoftware/newserv#425]
**Describe the bug**
Attempting to create an Episode 3 game while on the proxy server on another newserv instance causes "Error: no set data table exists for this version"

This could be user error, as I was able to play Episode 3 on SCHTServ in the past.

Might be worth mentioning but I also deleted every single map/quest on the proxy instance, though I doubt this is an issue?

**To reproduce**
1. Use proxy server
2. Make a game

**Game version(s) (choose one or more of the following):**
GC Ep3

**Server log output**
```
I 18515 2024-03-08 17:08:01 - [Commands] Received from LinkedSession:BE4AD36C:client (version=GC_EP3 command=EC flag=04)
0000 | EC 04 30 00 00 00 00 00 00 00 00 00 09 45 74 65 |   0          Ete
0010 | 73 74 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | st              
0020 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 03 |                 
I 18515 2024-03-08 17:08:01 - [Commands] Received from LinkedSession:BE4AD36C:server (version=GC_EP3 command=C9 flag=00)
0000 | C9 00 14 01 B4 44 00 00 52 00 29 00 A2 71 1A B9 |      D  R )  q  
0010 | 12 81 8A C9 82 91 FA D9 F2 A1 6A E9 62 B1 DA F9 |           j b   
0020 | D2 C1 4A 09 42 D1 BA 19 B2 E1 2A 29 22 F1 9A 39 |   J B     *)"  9
0030 | 92 01 0A 49 02 11 7A 59 72 21 EA 69 E2 31 5A 79 |    I  zYr! i 1Zy
0040 | 52 41 CA 89 C2 51 3A 99 32 61 AA A9 A2 71 1A B9 | RA   Q: 2a   q  
0050 | 12 81 8A C9 82 91 FA D9 F2 A1 6A E9 62 B1 DA F9 |           j b   
0060 | D2 C1 4A 09 42 D1 BA 19 B2 E1 2A 29 22 F1 9A 39 |   J B     *)"  9
0070 | 92 01 0A 49 02 11 7A 59 72 21 EA 69 E2 31 5A 79 |    I  zYr! i 1Zy
0080 | 52 41 CA 89 C2 51 3A 99 32 61 AA A9 A2 71 1A B9 | RA   Q: 2a   q  
0090 | 12 81 8A C9 82 91 FA D9 F2 A1 6A E9 62 B1 DA F9 |           j b   
00A0 | D2 C1 4A 09 42 D1 BA 19 B2 E1 2A 29 22 F1 9A 39 |   J B     *)"  9
00B0 | 92 01 0A 49 02 11 7A 59 72 21 EA 69 E2 31 5A 79 |    I  zYr! i 1Zy
00C0 | 52 41 CA 89 C2 51 3A 99 32 61 AA A9 A2 71 1A B9 | RA   Q: 2a   q  
00D0 | 12 81 8A C9 82 91 FA D9 F2 A1 6A E9 62 B1 DA F9 |           j b   
00E0 | D2 C1 4A 09 42 D1 BA 19 B2 E1 2A 29 22 F1 9A 39 |   J B     *)"  9
00F0 | 92 01 0A 49 02 11 7A 59 72 21 EA 69 E2 31 5A 79 |    I  zYr! i 1Zy
0100 | 52 41 CA 89 C2 51 3A 99 32 61 AA A9 A2 71 1A B9 | RA   Q: 2a   q  
0110 | 12 81 8A C9                                     |                 
I 18515 2024-03-08 17:08:01 - [ProxyServer:LinkedSession:BE4AD36C] The preceding command from the server was modified in transit
I 18515 2024-03-08 17:08:01 - [Commands] Sending to LinkedSession:BE4AD36C:client (version=GC_EP3 command=C9 flag=00)
0000 | C9 00 14 01 B4 44 00 00 52 00 00 00 00 00 00 00 |      D  R       
0010 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0020 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0030 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0040 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0050 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0060 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0070 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0080 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0090 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0100 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0110 | 00 00 00 00                                     |                 
I 18515 2024-03-08 17:08:01 - [Commands] Received from LinkedSession:BE4AD36C:server (version=GC_EP3 command=64 flag=01)
0000 | 64 01 84 11 00 00 00 00 00 00 00 00 00 00 00 00 | d               
0010 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0020 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0030 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0040 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0050 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0060 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0070 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0080 | 00 00 00 00 00 00 01 00 6C D3 4A BE 7F 00 00 01 |         l J     
0090 | 00 00 00 00 4D 61 74 74 00 00 00 00 00 00 00 00 |     Matt        
00A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
00F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0100 | 00 00 00 00 00 00 01 00 00 09 06 00 85 0F 30 08 |               0 
0110 | FF 00 00 00 05 00 00 01 01 00 00 00 08 00 00 00 |                 
0120 | 00 01 00 00 00 00 00 00 00 00 00 00 00 00 01 00 |                 
0130 | 00 00 00 00 01 00 00 00 08 00 00 00 01 01 00 00 |                 
0140 | 00 00 00 00 00 00 00 00 01 00 01 00 00 00 00 00 |                 
0150 | 01 00 00 00 08 00 00 00 02 00 05 00 F4 01 00 00 |                 
0160 | 00 00 00 00 02 00 01 00 12 00 00 28 01 00 00 00 |            (    
0170 | 00 00 00 00 03 00 00 00 00 04 00 00 00 00 00 00 |                 
0180 | 03 00 01 00 00 00 00 00 01 00 00 00 00 00 00 00 |                 
0190 | 03 01 00 00 00 04 00 00 00 00 00 00 04 00 01 00 |                 
01A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
01B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
01C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
01D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
01E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
01F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0200 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0210 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0220 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0230 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0240 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0250 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0260 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0270 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0280 | 00 00 00 00 00 00 00 03 00 00 00 00 00 00 00 00 |                 
0290 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
02F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0300 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0310 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0320 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0330 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0340 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0350 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0360 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0370 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0380 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0390 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
03F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0400 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0410 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0420 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0430 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0440 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0450 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0460 | 23 00 00 00 23 00 16 00 12 00 23 00 0A 00 28 00 | #   #     #   ( 
0470 | 00 00 00 00 00 00 20 41 1D 00 00 00 00 00 00 00 |        A        
0480 | 00 00 00 00 4D 61 74 74 00 00 00 00 00 00 00 00 |     Matt        
0490 | 00 00 00 00 00 00 00 00 00 00 00 00 6E F5 DF FF |             n   
04A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
04B0 | 00 00 00 00 06 02 00 02 31 00 00 00 00 00 00 00 |         1       
04C0 | 00 00 04 00 00 00 00 00 00 00 00 00 AB AA AA 3E |                >
04D0 | 00 00 00 00 00 00 00 00 01 00 00 00 02 00 00 00 |                 
04E0 | 02 01 00 00 04 00 00 00 01 00 00 00 00 00 00 00 |                 
04F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0500 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0510 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0520 | FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF |                 
0530 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0540 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0550 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0560 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0570 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0580 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0590 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
05A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
05B0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
05C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
05D0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
05E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
05F0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0600 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0610 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0620 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0630 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0640 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0650 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0660 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0670 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0680 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0690 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
06A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
06B0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
06C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
06D0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
06E0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
06F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0700 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0710 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0720 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0730 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0740 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0750 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0760 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0770 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0780 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0790 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
07A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
07B0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
07C0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
07D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
07E0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
07F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0800 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0810 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0820 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0830 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0840 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0850 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0860 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0870 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0880 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0890 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
08A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
08B0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
08C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
08D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
08E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
08F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0900 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0910 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0920 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0930 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0940 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0950 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0960 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0970 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0980 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0990 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
09A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
09B0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
09C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
09D0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
09E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
09F0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A00 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0A10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A20 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0A30 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A40 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0A50 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A60 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A70 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0A80 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0A90 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0AA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0AB0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0AC0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0AD0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0AE0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0AF0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B00 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0B10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B20 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0B30 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B40 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B50 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0B60 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B70 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0B80 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0B90 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0BA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0BB0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0BC0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0BD0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0BE0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0BF0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C00 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0C10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C20 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C30 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0C40 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C50 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0C60 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C70 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0C80 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0C90 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0CA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0CB0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0CC0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0CD0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0CE0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0CF0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D00 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D20 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D30 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D40 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D50 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D60 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D70 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D80 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0D90 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0DA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0DB0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0DC0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0DD0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0DE0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0DF0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E00 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0E10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E20 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0E30 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E40 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0E50 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E60 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E70 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0E80 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0E90 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0EA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0EB0 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0EC0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0ED0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0EE0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0EF0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F00 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0F10 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F20 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0F30 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F40 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F50 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0F60 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F70 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0F80 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0F90 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
0FA0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0FB0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0FC0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
0FD0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
0FE0 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
0FF0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1000 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
1010 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1020 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1030 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
1040 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1050 | 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 |                 
1060 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1070 | 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 |                 
1080 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1090 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
10A0 | 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF |                 
10B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
10C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
10D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
10E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
10F0 | FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1100 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1110 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1120 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1130 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1140 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1150 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1160 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1170 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 |                 
1180 | 00 00 00 00                                     |                 
I 18515 2024-03-08 17:08:01 - [ProxyServer:LinkedSession:BE4AD36C] Cleared lobby players
E 18515 2024-03-08 17:08:01 - [ProxyServer:LinkedSession:BE4AD36C] Failed to process command: no set data table exists for this version
I 18515 2024-03-08 17:08:01 - [Commands] Sending to LinkedSession:BE4AD36C:client (version=GC_EP3 command=11 flag=00)
0000 | 11 00 74 00 00 00 00 00 00 00 00 00 59 6F 75 20 |   t         You 
0010 | 63 61 6E 6E 6F 74 20 72 65 74 75 72 6E 0A 74 6F | cannot return to
0020 | 20 09 43 36 6E 65 77 73 65 72 76 09 43 37 0A 77 |   C6newserv C7 w
0030 | 68 69 6C 65 20 69 6E 20 61 20 67 61 6D 65 2E 0A | hile in a game. 
0040 | 0A 45 72 72 6F 72 3A 20 6E 6F 20 73 65 74 20 64 |  Error: no set d
0050 | 61 74 61 20 74 61 62 6C 65 20 65 78 69 73 74 73 | ata table exists
0060 | 20 66 6F 72 20 74 68 69 73 20 76 65 72 73 69 6F |  for this versio
0070 | 6E 00 00 00                                     | n               
```

**Comment by fuzziqersoftware** [comment]
This has been fixed by b491a57f. However, the maps are probably needed for other versions, even if you don't allow the drop mode to be changed - the proxy always loads maps for any games you join, since it allows the drop mode to be changed at any time. I suppose I could change this behavior, but it would make the code more complicated.

**Comment by Matt-Swift** [comment]
Oh I didn't remove the actual map maps, I just meant the Episode 3 map/quest files. Still, this is working now, thank you. :pray:
